test/incogniaApi.test.ts

```diff
@@ -126,27 +125,27 @@ describe('Incognia API', () => {
         riskAssessment: 'low_risk'
       }
 
-      const sessionToken = 'session_token'
+      const requestToken = 'request_token'
 
       nock(BASE_ENDPOINT)
         .post(`/v2/onboarding/signups`, {
-          session_token: sessionToken
+          request_token: requestToken
         })
         .reply(200, apiResponse)
 
       const webSignup = await IncogniaApi.registerWebSignup({
-        sessionToken
+        requestToken
       })
       expect(webSignup).toEqual(expectedResponse)
     })
 
     it('validates login', async () => {
       expect(() =>
         IncogniaApi.registerLogin({ accountId: 'id' } as any)
-      ).rejects.toThrow('Missing required parameter: installationId')
+      ).rejects.toThrow('Missing required parameter: requestToken')
       expect(() =>
-        IncogniaApi.registerLogin({ installationId: 'id' } as any)
-      ).rejects.toThrow('Missing required parameter: accountId')
+        IncogniaApi.registerLogin({ requestToke: 'token' } as any)
+      ).rejects.toThrow('Missing required parameter: requestToken')
```

Comment:
I think it should be like that, rigjt? To validate missing `accountId`
```suggestion
        IncogniaApi.registerLogin({ requestToken: 'token' } as any)
      ).rejects.toThrow('Missing required parameter: accountId')
```

Reply:
Oxe, errei total aqui. Nice catch.